packages/react/src/reconciler/renderer.ts

```diff
@@ -7,17 +7,13 @@ import { _render } from "./reconciler"
 /**
  * @deprecated Use `createRoot(renderer).render(node)` instead
  */
-export async function render(node: ReactNode, rendererConfig: CliRendererConfig = {}): Promise<void> {
+export async function render(
+  node: ReactNode,
+  rendererConfig: CliRendererConfig = {},
+): Promise<{ renderer: CliRenderer }> {
   const renderer = await createCliRenderer(rendererConfig)
-  engine.attach(renderer)
```

Comment:
We somehow still need to get this attached for now, as animations with useTimeline won't work otherwise. 

This is another outstanding refactor, because the engine is currently global, but needs to be per renderer to support multiple renderers in the same process.